src/test/broadcastprofile_test.cpp

```diff
@@ -42,11 +42,10 @@ TEST(BroadcastProfileTest, SaveAndLoadXML) {
     // Preliminary: set a discriminating value in one of the profile fields
     QString streamName("unit testing in progress");
 
-    BroadcastProfile profile("Unit Testing Profile");
+    BroadcastProfile profile("Broadcast Profile test");
     profile.setStreamName(streamName);
 
-    QString filename = profile.getProfileName() + QString(".bcp.xml");
-
+    QString filename = QDir::temp().filePath(profile.getProfileName() + QString(".bcp.xml"));
     // Call save() on a profile and assert it actually exists
     QFile::remove(filename); // First, make sure it doesn't exists
```

Comment:
Use `QTemporaryDir` instead. That way the files are automatically removed when `tempDir` goes out of scope and we don't clutter the users `/tmp` directory. That way we also don't need to try to remove the file first.

```suggestion
    QTemporaryDir tempDir;
    QString filename = tempDir.filePath(profile.getProfileName() + QString(".bcp.xml"));
    // Call save() on a profile and assert it actually exists
```